### PR TITLE
fix error body is not JSON but text (is it always?)

### PR DESCRIPTION
**Pull request by lexoyo**

### CODE REVIEW

**Comment by lexoyo** [comment]
maybe we should remove the JSON.parse completely? I don't know how dropbox is supposed to send errors

File: lib/unifile-dropbox.js [hunk 1]
```diff
@@ -51,10 +51,13 @@ function callAPI(session, path, data, subdomain = 'api', isJson = true, headers
 			}
 			if(res.statusCode >= 400) {
 				return reject({statusCode: res.statusCode, message: (() => {
-					console.error('Error in unifile', body.toString());
+					const bodyStr = body.toString();
```
Comment (JbIPS) [inline lib/unifile-dropbox.js]: Could you test if body is already an object and if it has a property named `error_summary`. If it does, return it, if not return as-is

Comment (lexoyo) [reply lib/unifile-dropbox.js]: Yes ok
I will remove the try/catch then

Comment (lexoyo) [reply lib/unifile-dropbox.js]: as we discussed the try catch is needed 
should I merge?

Comment (JbIPS) [reply lib/unifile-dropbox.js]: Actually the body is `JSON` when `isJson === true` . So we should test its value.

**Comment by JbIPS** [comment]
I'm closing this since I solved it in https://github.com/silexlabs/unifile/commit/b4abca76581aa8d0473f40d348c12ee0bae6bb54

**Pull request closed without merging**